.circleci/config.yml

```diff
@@ -146,7 +146,6 @@ workflows:
           name: dev-promote-patch
           orb-name: netgaintechnology/build-orb
           ssh-fingerprints: 9d:71:05:ff:1f:30:b2:c4:0b:9e:27:a9:09:d8:d0:c1
-          cleanup-tags: true
```

Comment:
these are needed to cleanup the tags circleci creates to trigger the integration workflow.

Reply:
I’m looking into this now (via mobile) to try and see what changed / caused this to break.

Reply:
Found it. Below is the PR where this was merged into prod by CircleCI.
https://github.com/CircleCI-Public/orb-tools-orb/pull/64
Notes from RP:
```
cleanup-tags isn't unused. it's a boolean for whether or not to do so. the cleanup-tags-pattern param then allows you to specify which tags to cleanup (if cleanup-tags is true). however, it is unused in the dev-promote-prod job in particular, where tags are always cleaned up & the user specifies only which tags should be removed. thank you !
```